src/docs/asciidoc/http-server.adoc

```diff
@@ -685,3 +685,108 @@ HTTP/2 200
 hello
 ----
 ====
+
+== Metrics
+HTTP server supports built-in integration with https://micrometer.io/[`Micrometer`].
+It exposes all metrics with prefix `reactor.netty.http.server`.
+
+The following table provides information for the HTTP server metrics:
+
+[width="100%",options="header"]
+|=======
+| metric name | type | description
+| reactor.netty.http.server.data.received | DistributionSummary | Amount of the data that is received, in bytes
```

Comment:
You can remove "that is" in all of these (as in a previous comment, it's OK to be more telegraphic in tables).